code/modules/antagonists/cult/cult_items.dm

```diff
@@ -1005,3 +1007,13 @@
 					throw_at(D.thrower, 7, 1, D.thrower)
 	else
 		..()
+
+obj/item/shield/mirror/proc/break_mirror(turf/T) //yogs mirror proc start
```

Comment:
Donk this proc into ``yogstation/code/modules/antagonists/cult/cult_items.dm``

Reply:
mk